solution.cpp

```diff
@@ -1,23 +1,89 @@
 /* ----------------------------------------------------------------------------
- * Copyright &copy; 2015 TODO(your name) <TODO(your email)>
+ * Copyright &copy; 2015 Jasem Altaweel <lwa@csu.fullerton.edu>
  * Released under the [MIT License] (http://opensource.org/licenses/MIT)
  * ------------------------------------------------------------------------- */
 
-/**
- * TODO: program description
- */
-
-// TODO: `#include`s and function prototypes
-
-// ----------------------------------------------------------------------------
-
-int main() {
-    // TODO
-
-    return 0;  // success
-}
-
-// ----------------------------------------------------------------------------
-
-// TODO: other functions
-
++ * Copyright &copy; 2015 James Paul Mabe <jamespmabe@gmail.com>
```

Comment:
best to also include an explanatory note about who wrote what code, especially if the previous author wrote most of it.  but this is fine -- thanks for giving credit where credit is due :-)